docs/manual/output.md

```diff
@@ -91,3 +88,212 @@ expressed in the typical physical unit symbols. Unfortunately, column names
+`/RMG/Output/Germanium/StoreTrackID true`
+
+As mentioned earlier output schemes also provide a mechanism for filtering
+events. One useful option is to only write out events in which energy
+was deposited in a Germanium detector. This is used since the
+other detector systems (LAr, water Cherenkov etc.) often act a "vetos's",
+so we are not interested in the energy deposited or steps if an event in the
+Germanium did not occur.
+The macro commands:
+
+`/RMG/Output/Germanium/AddDetectorForEdepThreshold {DET_UID}`
+`/RMG/Output/Germanium/SetEdepCutLow {ELOW}`
+`/RMG/Output/Germanium/SetEdepCutHigh {EHIGH}`
+
+implement this functionality, for every event the total energy deposited is
+computed. This is based on summing the energy deposited in each `{DET_UID}` added,
+or across all registered sensitive Germanium detectors (if this macro command is not used).
+The event is then discarded if the energy is less than or equal to `ELOW` or less than `EHIGH`.
+
+:::{note}
+This mechanism will remove the data from the event across all output schemes, not only the Germanium!
+:::
+
+Similarly, for simulations involving optical photons it is possible to discard all optical
+photon tracks before simulating them if no energy was deposited in Germanium. This can be enabled
+with the:
+
+`/RMG/Output/Germanium/DiscardPhotonsIfNoGermaniumEdep`
+
+macro command.
+
+By default, the position saved for each step is the average of the pre and post-step point. This can
+be controlled with the macro command:
+
+`/RMG/Output/Germanium/StepPositionMode`
+
+which can be set to `Average` (the default), `Both` (saves) also the pre and post steps, or `Pre`/`Post`.
+
+:::{important}
+For gammas the position saved is always that of the post-step, since all gamma interactions are discrete.
+:::
+
+Typically only steps where some energy was deposited are written out to disk, to control
+this behaviour there is a macro command:
+
+`/RMG/Output/Germanium/DiscardZeroEnergyHits`
+
+Finally, it is possible to "pre-cluster" the steps, this is used to reduce the
+amount of data written out to disk by combining steps very close together.
+Since the surface region of a HPGe detector has different properties to the bulk
+this clustering can be performed differently for surface and bulk hits
+(see [data-reduction](#data-reduction-methods) for more details).
+
+## Scintillator detectors
+
+This output scheme is used to record the steps in scintillation detectors (typically LAr),
+this is a calometric approach recording the energy deposited and steps. While the Optical
+output scheme is instead used for recording the detected optical photons.
+Most functionality is similar to the Germanium output scheme with a few exceptions.
+
+- Unlike for Germanium detectors the distance to the detector surface is not calculated,
+- The stacking possibility for optical tracks is not implemented,
+- The velocity of the particles can be saved using the
+
+`/RMG/Output/Scintillator/StoreParticleVelocities`
```

Comment:
here and in all the rest: put everything on the same line and link to the macro command reference